mobile/src/components/common/pagination/pagination.tsx

```diff
@@ -12,6 +12,8 @@ type Props = {
   onPageChange: (page: number) => void;
 };
 
+const hitSlop = { top: 20, bottom: 20, left: 20, right: 20 };
```

Comment:
If it's not a global variable, move this inside the component

Reply:
ok